travis/test_server.py

```diff
@@ -136,8 +137,13 @@ def get_addons_path(travis_dependencies_dir, travis_build_dir, server_path):
     return addons_path
 
 
-def get_server_script(odoo_version):
-    return 'odoo-bin' if float(odoo_version) >= 10 else 'openerp-server'
+def get_server_script(server_path):
+    try:
+        shutil.copy(os.path.join(server_path, 'openerp-server'),
+                    os.path.join(server_path, 'odoo-bin'))
+    except IOError:
```

Comment:
@moylop260 
What do you think about the previous comment?
I do not remember where PR agreed this change